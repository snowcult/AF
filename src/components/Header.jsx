import React, { memo } from 'react';

const Header = memo(() => {
  return (
    <div>
      <h1 className="tc f-2 white">AiFriends</h1>
    </div>
  );
});
Header.displayName = 'Header';
export default Header;
