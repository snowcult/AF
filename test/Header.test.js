import { shallow } from 'enzyme';
import React from 'react';
import Header from '../src/components/Header.jsx';

it('expect to render Header comp', () => {
  expect(shallow(<Header />)).toMatchSnapshot();
});
