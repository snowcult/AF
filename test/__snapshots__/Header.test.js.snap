// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Header comp 1`] = `
<div>
  <h1
    className="tc f-2 white"
  >
    AiFriends
  </h1>
</div>
`;
